### PR TITLE
Aida 530

### PR DESCRIPTION
**Pull request by sharknc**
- Documented all of aifutils.py using reStructuredText docstrings
- Required installation of Sphinx in setup.py 
- Ran Sphinx setup and created all appropriate files needed to generate the python documentation
- Created update_documentation.sh script that can be run to generate the latest python documentation
- Updated README.md to include documentation on how to generate both Javadoc and python documentation
- Updated .gitignore to ignore any documentation that was generated in python of Java. 

**Important**
You will need to install Sphinx in your virtual environment using pip install in order to run the update_documentation.sh script.

### CODE REVIEW

**Comment by jakevb** [comment]
When no _static folder is in the python/docs/source/ folder, Sphinx gives a warning of 
copying static files... WARNING: html_static_path entry '/AIDA-Interchange-Format/python/docs/source/_static' does not exist.  However, documentation seems to be generated correctly.

**Comment by jakevb** [comment]
If you could also update the documentation to the mark_ldc_time method which was recently added to master, that would be helpful.

**Comment by sharknc** [comment]
> When no _static folder is in the python/docs/source/ folder, Sphinx gives a warning of
> copying static files... WARNING: html_static_path entry '/AIDA-Interchange-Format/python/docs/source/_static' does not exist. However, documentation seems to be generated correctly.

I have updated the config.py file to no longer require this directories. I have also removed these directories from the repo. The script should now work without them.

**Comment by sharknc** [comment]
> When no _static folder is in the python/docs/source/ folder, Sphinx gives a warning of
> copying static files... WARNING: html_static_path entry '/AIDA-Interchange-Format/python/docs/source/_static' does not exist. However, documentation seems to be generated correctly.

**Comment by sharknc** [comment]
> If you could also update the documentation to the mark_ldc_time method which was recently added to master, that would be helpful.

I updated the documentation for mark_ldc_time. I also updated ldc_Time to be renamed to ldc_time.

File: python/aida_interchange/aifutils.py [hunk 1]
```diff
@@ -522,40 +899,49 @@ def _make_aif_justification(g, doc_id, class_type, system, confidence,
...
     """
-    Get all confidence structures associated with an AIF object.
+    Retrieve all confidence assertions from an entity.
 
     This does not get confidences attached to sub-graphs containing the object.
 
-    :return: A list of confidence assertions describing this object.
-  """
+    :param rdflib.graph.Graph g: The underlying RDF model
+    :param rdflib.term.URIRef confidenced_object: he entity from which to retrieve
```
Comment (jakevb) [inline python/aida_interchange/aifutils.py]: Typo: "the" not "he"

File: python/aida_interchange/aifutils.py [hunk 2]
```diff
@@ -408,27 +700,41 @@ def mark_informative_justification(g, resource, informative_justification):
...
     """
     Mark the given resources as mutually exclusive.
 
-    :param alternatives: a map from the collection of edges which form a sub-graph for
-    an alternative to the confidence associated with an alternative.
-    :param system: The system object for the system which contains the mutual exclusion
-    :param none_of_the_above_prob: if not None, the given confidence will be applied for
-    the "none of the above" option.
-    :return: The mutual exclusion assertion.
+    This is a special case of [markAsMutuallyExclusive] where the alternatives are
```
Comment (jakevb) [inline python/aida_interchange/aifutils.py]: Should this be mark_as_mutually_exclusive?

Comment (sharknc) [reply python/aida_interchange/aifutils.py]: yep

Comment (sharknc) [reply python/aida_interchange/aifutils.py]: fixed

File: python/aida_interchange/aifutils.py (hunk 1 above)
Comment (sharknc) [reply python/aida_interchange/aifutils.py]: fixed

**Pull request merged**